README.md

```diff
@@ -1,16 +1,435 @@
 # eventsourcingdb
 
-The official Rust client SDK for [EventSourcingDB](https://www.eventsourcingdb.io/).
+The official Rust client SDK for [EventSourcingDB](https://www.eventsourcingdb.io) – a purpose-built database for event sourcing.
 
-# Project status
+EventSourcingDB enables you to build and operate event-driven applications with native support for writing, reading, and observing events. This client SDK provides convenient access to its capabilities in Rust.
 
-This is a work in progress and not yet ready for production use.
-Based on the [compliance criteria](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/) the SDK covers these criteria:
+For more information on EventSourcingDB, see its [official documentation](https://docs.eventsourcingdb.io/).
 
-- 🚀 [Essentials](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#essentials)
-- 🚀 [Writing Events](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#writing-events)
-- 🚀 [Reading Events](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#reading-events)
-- 🚀 [Using EventQL](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#using-eventql)
-- 🚀 [Observing Events](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#observing-events)
-- 🚀 [Metadata and Discovery](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#metadata-and-discovery)
-- 🚀 [Testcontainers Support](https://docs.eventsourcingdb.io/client-sdks/compliance-criteria/#testcontainers-support)
+This client SDK includes support for [Testcontainers](https://testcontainers.com/) to spin up EventSourcingDB instances in integration tests. For details, see [Using Testcontainers](#using-testcontainers).
+
+## Getting Started
+
+Install the client SDK:
+
+```shell
+cargo add eventsourcingdb
+```
+
+Import the package and create an instance by providing the URL of your EventSourcingDB instance and the API token to use:
+
+```rust
+use eventsourcingdb::client::Client;
+
+// ...
+
+let base_url: Url = "localhost:3000".parse().unwrap();
+let api_token = "secret";
+let client = Client::new(base_url, api_token);
+```
+
+Then call the `ping` function to check whether the instance is reachable. If it is not, the function will return an error:
```

Comment:
`ping` -> [`client::Client::ping`] for RustDoc (so it links). This would also not be necessary when separating the two docs parts.